examples/eks-efa/README.md

```diff
@@ -0,0 +1,659 @@
+# EKS Blueprint Example with Elastic Fabric Adapter
```

Comment:
this directory no longer exists on `main` - it was converted into `patterns/nvidia-gpu-efa` so we can remove everything under this `examples/eks-efa`

Reply:
removed examples directory